src/styles/variables.scss

```diff
@@ -0,0 +1,21 @@
+$day-size: 100px;
+$gap-between: 1px;
+$calendar-width: (7 * $day-size) + 20px + $gap-between * 6;
+$day-text-color: black;
+$day-background-color: #eee;
+$day-border: 1px solid black;
+$day-text-size: 30px;
+$day-font: Arial, Helvetica, sans-serif;
+$start-day-modifiers: (mon, tue, wed, thu, fri, sat, sun);
+$month-length-modifiers: (28, 29, 30, 31);
+$box-with-gap: $day-size + $gap-between;
+$hover-color-day: #ffbfcb;
+$days-of-the-week: (
+  'mon' 0,
```

Comment:
There is a missing colon after 'mon'. It should be 'mon: 0'. This is an error related to the task requirements.